### ISSUE TITLE
mount /Users as root

### ISSUE THREAD

**Issue by mafintosh** [boot2docker/boot2docker#639]
Is there any way to mount the shared folder `/Users` as root instead of as the docker user?
Right now I log in using ssh and execute

```
sudo umount /Users
sudo mount -t vboxsf Users /Users/
```

But the change isn't persistent on restart etc.

**Comment by eerkel** [comment]
@paolomainardi Did you work out your Mac permissions issues? I'm having the same problem, where I cannot make a directory writeable by the wwwdata user, which is a fairly common requirement of web apps (e.g. Wordpress, Drupal, etc).

**Comment by mikeys** [comment]
@eerkel @paolomainardi I'm having the exact same problem. Did you manage to solve the problem by any chance?

**Comment by paolomainardi** [comment]
Yes you have to use NFS: https://github.com/boot2docker/boot2docker/issues/587#issuecomment-66935011

**Comment by eerkel** [comment]
@mikeys The performance problem is definitely solved by using NFS. However, I will say that after spending days on getting Docker working for my dev environment, I've gone back to the Mac hack stack. There are many hurdles. After this one I ran into permissions issues, so if you want to sync a volume to your local dev machine (i.e. Mac/Windows), and that volume is acted on by the container in the form of a chmod or chown, then you are going to have a blast. 

Docker feels like the future (for developers too), I just don't have time right now to spend so many days on something that is still emerging and that has minimal documentation pertinent to dev environments.
